.env.example

```diff
@@ -4,6 +4,9 @@ CF_ORG=dfe-teacher-services
 CF_SPACE=teaching-vacancies-dev
 CF_USERNAME=
 CF_PASSWORD=
+CF_POSTGRES_SERVICE_ORIGIN=
+CF_SPACE_ORIGIN=
+CF_POSTGRES_SERVICE_TARGET=
```

Comment:
You need `CF_SPACE_TARGET` too

Reply:
but only if you need to take the production dump and load it to `dev` or `staging`